.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: required
+cache:
+  directories:
+    - $HOME/.m2
+language: java
+jdk:
+  - oraclejdk8
+install: true
+script:
+  - '[ "${TRAVIS_PULL_REQUEST}" != "false" ] || ./mvnw package -Pfull -Pspring -U -Dmaven.test.redirectTestOutputToFile=true'
+  - '[ "${TRAVIS_PULL_REQUEST}"  = "false" ] || ./mvnw package -Pfull -Pspring -U -Dmaven.test.redirectTestOutputToFile=true'
```

Comment:
What is the logic behind this mutually exclusive commands?
Looks like they are the same, and one or another will be executed independently of the `TRAVIS_PULL_REQUEST` value.
I mean there is just enough to have:
```
script:
./mvnw package -Pfull -Pspring -U -Dmaven.test.redirectTestOutputToFile=true'
```

I won't mind if I miss something, but at least, please, explain how this should work.

Thanks